e2e/vitest.test.js

```diff
@@ -127,6 +127,23 @@ describe('examples/vitest', () => {
 		})
 	}, 10000) // 10s timeout
 
+	test('it handles no location being present', (done) => {
+		exec('npm test -- --config vitest.override.config.js', { cwd, env }, (error, stdout, stderr) => {
```

Comment:
```suggestion
		exec('npm test -- --includeTaskLocation false', { cwd, env }, (error, stdout, stderr) => {
```
We can also do this so we don't need to create new vitest config for test.